src/core/usecases/complete-question.usecase.ts

```diff
@@ -0,0 +1,89 @@
+import { UseCase } from '../base/use-case';
+import { GameModuleRepository } from '../domain/repository/game-module.repository';
+import { ProgressionRepository } from '../domain/repository/progression.repository';
+import { McqModule } from '../domain/model/McqModule';
+import { Progression, ProgressionType } from '../domain/model/Progression';
+import { QuestionNotFoundError } from '../domain/error/QuestionNotFoundError';
+import { InvalidAnswerError } from '../domain/error/InvalidAnswerError';
+
+export type CompleteQuestionCommand = {
+  userId: string;
+  questionId: string;
+  answer: {
+    selectedChoiceId: string;
+  };
+};
```

Comment:
Du coup comme on ne manipule pas des "questions", faut modif ça, tu peux faire plus ou moins comme la création. 
```suggestion
export type CompleteGameModuleCommand = {
  userId: string;
  moduleId: string;
  mcq?: {
    choiceId: string;
  };
};
```